### PR TITLE
Added username to Acess Token

### PR DESCRIPTION
**Pull request by Bogdan-Hasanov**

### CODE REVIEW

File: OutOfSchool/OutOfSchool.WebApi/appsettings.json [hunk 1]
```diff
@@ -9,5 +9,8 @@
   "Identity": {
     "Authority": "http://localhost:5443"
   },
+  "ConnectionStrings": {
```
Comment (dmitrykiev) [inline OutOfSchool/OutOfSchool.WebApi/appsettings.json:12]: is this change needed?

**Review by dmitrykiev — APPROVED** [review]

**Review by DmyMi — APPROVED** [review]
